dist/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" 
+    integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>To Do List</title>
+<script defer src="main.js"></script></head>
+<body>
+    <div class="container">
+        <div class="header">
+            <h1 class="title">Today's todo list</h1>
+            <button class="head-btn"><i class="fa">&#xf021;</i>   </button> 
+        </div><hr>
+        <form id="inputForm" class="input-field">
+            <input type="text" id="inputField" class="input" placeholder="Add to your list..."><button id="addTodo" class="btn"><i class='fas fa-arrow-left'></i></button>
+        </form><hr>
+        <ul id="todoList" class="my-todos">         
+        </ul>
+        <div>
+           <button id="clear" class="clr-btn"><p>Clear all complated</p></button>
+        </div>  
+    </div>
```

Comment:
- [ ] @ezraermy great work on your user interface, however, the buttons are not positioned correctly, kindly check the comparison below:

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/225039502-7bffb837-81ab-4ece-b0be-79d691100dcb.png)  |  ![image](https://user-images.githubusercontent.com/61408860/225039560-b64325ac-2a4b-46ba-b14a-46174adfdb71.png)